ppr-ui/package.json

```diff
@@ -25,7 +25,7 @@
     "http-status-codes": "^2.1.4",
     "material-design-icons": "^3.0.1",
     "moment": "^2.29.1",
-    "sbc-common-components": "2.4.19",
+    "sbc-common-components": "2.4.33",
```

Comment:
This substantial change will require some additional testing around the components used in this project.

Due to a lack of release notes, the tester should look at the commit history between v2.4.19 and v2.4.33 to identify potential breakages.

Reply:
I will make a note to QA.

Reply:
Ok. (I do not have permissions to resolve this conversation.)